vtu11/impl/vtu11_impl.hpp

```diff
@@ -230,14 +230,16 @@ void parallelWrite( const std::string& path,
 {
   fs::path directory( path );
 
-  if( !fs::exists( directory ) )
+  // create base directory for all files if not existing
+  if( !fs::exists( directory ) && fileId == 0 )
```

Comment:
I think here we should add `&& !path.empty( )`. Because if I'm not mistaken creating a directory from an empty string doesn't work.

I did this already in a separate branch, but we can add it here.

Reply:
makes sense, can you add what you have?